Also add permission change. (#20029)
DataDog/documentation: content/en/database_monitoring/guide/pg15_upgrade.md

```yaml
ssl: allow
```

Restart the agent after applying this change.

> The `datadog` user needs an additional permission enabled. Execute this statement on each DB host:

Reviewer: ```suggestion
Run this command on each database host to enable the additional permission needed for the `datadog` user:
```

Author: Thanks, just pushed that change.
